### DISCUSSION TITLE
IO-Broker MQTT

### DISCUSSION THREAD

**Discussion by stmb-100** [emsesp/EMS-ESP32#1052] [General]
Does anyone also have the problem that no values are displayed in IO broker. 

Configuration MQTT EMS-ESP
![grafik](https://user-images.githubusercontent.com/54959405/121900082-4fb9e900-cd25-11eb-850c-72fa6abeb614.png)

![grafik](https://user-images.githubusercontent.com/54959405/121900305-94de1b00-cd25-11eb-9469-ab1546db011f.png)

![grafik](https://user-images.githubusercontent.com/54959405/121900389-aaebdb80-cd25-11eb-8aec-c1338c7daa0c.png)

**Comment by stmb-100** [comment]
![grafik](https://user-images.githubusercontent.com/54959405/121900954-47ae7900-cd26-11eb-9519-ac4524f1d653.png)

**Comment by MichaelDvP** [comment]
These are command subscriptions only and do not publish values. The values are published as json in the `_data` topics.
I have a [experimental version](https://github.com/emsesp/EMS-ESP32/files/6649049/firmware.zip) that also publishes single values on change, or you can use the the api with the [ioBroker-Adapter from tp1de](https://github.com/tp1de/ioBroker.ems-esp).

**Comment by proddy** [comment]
@stmb-100 close this?
